extensions/wikia/AdEngine/js/lookup/prebid/adapters/audienceNetwork.js

```diff
@@ -0,0 +1,70 @@
+/*global define*/
+define('ext.wikia.adEngine.lookup.prebid.adapters.audienceNetwork',[
+	'ext.wikia.adEngine.context.slotsContext',
+	'wikia.geo',
+	'wikia.instantGlobals',
+	'ext.wikia.adEngine.adContext'
+], function (slotsContext, geo, instantGlobals, adContext) {
+	'use strict';
+
+	var bidderName = 'audienceNetwork',
+		slots = {
+			mercury: {
+				MOBILE_TOP_LEADERBOARD: {
+					sizes: [
+						[320, 50]
+					],
+					placementId: '963689110376230_1245837502161388'
+				},
+				MOBILE_IN_CONTENT: {
+					sizes: [
+						[300, 250],
+						[320, 480]
+					],
+					placementId: '963689110376230_1245838625494609'
+				},
+				MOBILE_PREFOOTER: {
+					sizes: [
+						[300, 250],
+						[320, 50]
+					],
+					placementId: '963689110376230_1245839585494513'
+				}
+			}
+		};
+
+	function isEnabled() {
+		// TODO: add and set and check here instantGlobals.wgAdDriverAudienceNetworkBidderCountries
```

Comment:
create follow up story for production ready code and add this as a subtask / AC please